lib/fastlane/plugin/instrumented_tests/actions/instrumented_tests_action.rb

```diff
@@ -61,9 +59,9 @@ def self.create_emulator(params)
       def self.start_emulator(params)
         UI.important("Starting AVD...")
         ui_args = "-gpu on"
-        ui_args >> "-no-window" if params[:avd_hide]
-        ui_args >> params[:emulator_options] if params[:emulator_options] != nil
-        start_avd = ["#{params[:sdk_path]}/tools/emulator", "-avd #{params[:avd_name]}", "#{ui_args}", "-port #{params[:emulator_port]}" ].join(" ")
+        ui_args << " -no-window" if params[:avd_hide]
+        ui_args << " " << params[:emulator_options] if params[:emulator_options] != nil
```

Comment:
maybe do `ui_args << " #{params[:emulator_options]}"  if params[:emulator_options] != nil`
a little easier to read

Reply:
Good point 👍